tests/unit/BeginFromCurrentStateTests.swift

```diff
@@ -240,10 +240,12 @@ class BeginFromCurrentStateTests: XCTestCase {
 
     XCTAssertEqual(addedAnimations.count, 2)
 
-    let animation = addedAnimations.last as! CABasicAnimation
-    XCTAssertFalse(animation.isAdditive)
-    XCTAssertEqual(animation.keyPath, AnimatableKeyPath.opacity.rawValue)
-    XCTAssertEqual(animation.fromValue as! Float, initialValue)
-    XCTAssertEqualWithAccuracy(animation.toValue as! CGFloat, 1.0, accuracy: 0.0001)
+    if addedAnimations.count == 2 {
+      let animation = addedAnimations.last as! CABasicAnimation
+      XCTAssertFalse(animation.isAdditive)
+      XCTAssertEqual(animation.keyPath, AnimatableKeyPath.opacity.rawValue)
+      XCTAssertEqual(animation.fromValue as! Float, initialValue)
+      XCTAssertEqualWithAccuracy(animation.toValue as! CGFloat, 1.0, accuracy: 0.0001)
+    }
```

Comment:
Should this be paired with an else block with an `XCTAssertTrue(false)`?

Reply:
This is paired with the XCTAssertEqual above. If that assertion fails, this logic won't be executed.